src/ray/raylet/agent_manager.h

```diff
@@ -88,17 +89,36 @@ class AgentManager : public rpc::AgentManagerServiceHandler {
   virtual void DeleteRuntimeEnvIfPossible(const std::string &serialized_runtime_env,
                                           DeleteRuntimeEnvIfPossibleCallback callback);
 
+  /// Try to Get the information about the agent process,
+  /// if successful will return `ray::Status::OK`, otherwise will return
```

Comment:
Can you Move this string to \return line not here?

Reply:
done.